astronomer/providers/amazon/aws/hooks/base_aws_async.py

```diff
@@ -5,7 +5,14 @@
 
 
 class AwsBaseHookAsync(AwsBaseHook):
-    """Interacts with AWS using aiobotocore asynchronously."""
+    """
+    Interacts with AWS using aiobotocore asynchronously.
+
+    Limitations: AwsBaseHookAsync uses aiobotocore to create asynchronous S3 hooks. Hence,
```

Comment:
Note sure limitation is sphinx directive or not please use the note 
.. note::
  your text

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/388/commits/be06536a3f6f9556a92f086364c7e299f19404af)